content/pt-br/community/static/cncf-code-of-conduct.md

```diff
@@ -13,32 +14,61 @@ todos, independentemente do nível de experiência, gênero, identidade e expres
 orientação sexual, deficiência, aparência pessoal, tamanho do corpo, raça, etnia, idade,
 religião ou nacionalidade.
 
-Exemplos de comportamento inaceitável pelos participantes incluem:
+## Escopo 
 
-* O uso de linguagem ou imagens sexualizadas
-* Ataques pessoais
-* Trolling ou insultar / comentários depreciativos
+Este código de conduta aplica-se tanto nos espaços do projeto como nos espaços públicos
+quando um indivíduo está representando o projeto ou sua comunidade.
+
+### Eventos CNCF
+
+Eventos da CNCF, ou eventos organizados pela Linux Foundation com equipe profissional de eventos, são regidos pelo [Código de Conduta da Linux Foundation](https://events.linuxfoundation.org/code-of-conduct/) disponível na página do evento. Isso foi projetado para ser utilizado em conjunto com o Código de Conduta do CNCF.
+
+## Nossos Padrões
+
+Exemplos de comportamento que contribuem para um ambiente positivo incluem:
+
+* Demonstrar empatia e gentileza para com outras pessoas
+* Ser respeitoso com as diferentes opiniões, pontos de vista e experiências
+* Dar e aceitar educadamente feedback construtivo
+* Aceitar a responsabilidade e pedir desculpas aos afetados por nossos erros,
+   e aprender com a experiência
+* Concentrar-se no que é melhor não apenas para nós como indivíduos, mas para a
+   comunidade em geral
+
+Exemplos de comportamento inaceitável incluem:
+
+* O uso de linguagem ou imagens sexualizadas, e atenção sexual ou
+   avanços de qualquer tipo
+* Trolling, comentários insultuosos ou depreciativos e ataques pessoais ou políticos
 * Assédio público ou privado
 * Publicação de informações privadas de outras pessoas, como endereços físicos ou eletrônicos,
  sem permissão explícita
-* Outra conduta antiética ou não profissional.
+* Outra conduta que possa razoavelmente ser considerada inadequada em um
+   ambiente profissional
 
 Os mantenedores do projeto têm o direito e a responsabilidade de remover, editar ou rejeitar
 comentários, commits, códigos, edições do wiki, questões e outras contribuições que não são
 alinhado com este Código de Conduta. Ao adotar este Código de Conduta, os mantenedores do projeto
 comprometer-se a aplicar de maneira justa e consistente esses princípios em todos os aspectos
```

Comment:
Typo: comprometer-se

Reply:
Fixed in [dff104a](https://github.com/kubernetes/website/pull/38407/commits/dff104a0783dd29768a20e87f57c8416c461c613). Thank you.